systemtest/src/test/java/io/strimzi/systemtest/operators/ReconciliationST.java

```diff
@@ -88,91 +92,92 @@ void testPauseReconciliationInKafkaAndKafkaConnectWithConnector(ExtensionContext
-        KafkaConnectorResource.replaceKafkaConnectorResource(clusterName, connector -> {
+        KafkaConnectorResource.replaceKafkaConnectorResourceInSpecificNamespace(clusterName, connector -> {
             connector.getMetadata().setAnnotations(PAUSE_ANNO);
             connector.getSpec().setTasksMax(SCALE_TO);
-        });
+        }, namespaceName);
 
-        KafkaConnectorUtils.waitForConnectorStatus(clusterName, CustomResourceStatus.ReconciliationPaused);
-        KafkaConnectorUtils.waitForConnectorSpecFromConnectAPIStability(connectPodName, clusterName, connectorSpec);
+        KafkaConnectorUtils.waitForConnectorStatus(namespaceName, clusterName, CustomResourceStatus.ReconciliationPaused);
+        KafkaConnectorUtils.waitForConnectorSpecFromConnectAPIStability(namespaceName, connectPodName, clusterName, connectorSpec);
 
         LOGGER.info("Setting annotation to \"false\", taskMax should be increased to {}", SCALE_TO);
-        KafkaConnectorResource.replaceKafkaConnectorResource(clusterName, connector ->
-            connector.getMetadata().getAnnotations().replace(Annotations.ANNO_STRIMZI_IO_PAUSE_RECONCILIATION, "true", "false"));
+        KafkaConnectorResource.replaceKafkaConnectorResourceInSpecificNamespace(clusterName, connector ->
+            connector.getMetadata().getAnnotations().replace(Annotations.ANNO_STRIMZI_IO_PAUSE_RECONCILIATION, "true", "false"), namespaceName);
 
         String oldConfig = new JsonObject(connectorSpec).getValue("config").toString();
-        JsonObject newConfig = new JsonObject(KafkaConnectorUtils.waitForConnectorConfigUpdate(connectPodName, clusterName, oldConfig, "localhost"));
+        JsonObject newConfig = new JsonObject(KafkaConnectorUtils.waitForConnectorConfigUpdate(namespaceName, connectPodName, clusterName, oldConfig, "localhost"));
 
         assertThat(newConfig.getValue("tasks.max"), is(Integer.toString(SCALE_TO)));
     }
 
-    @IsolatedTest
+    @ParallelNamespaceTest
     void testPauseReconciliationInKafkaRebalanceAndTopic(ExtensionContext extensionContext) {
-        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String namespaceName = extensionContext.getStore(ExtensionContext.Namespace.GLOBAL).get(Constants.NAMESPACE_KEY).toString();
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String topicName = mapWithTestTopics.get(extensionContext.getDisplayName());
 
         resourceManager.createResource(extensionContext, KafkaTemplates.kafkaWithCruiseControl(clusterName, 3, 3).build());
-
-        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(clusterName, TOPIC_NAME).build());
+        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(clusterName, topicName).build());
 
         /* TODO: after issue with TO and pausing reconciliations in KafkaTopic will be fixed, uncomment these lines
```

Comment:
Maybe this is something what should be enabled?

Reply:
Who knows :D

Reply:
discussed offline...